react/features/remote-video-menu/components/AbstractMuteRemoteParticipantDialog.js

```diff
@@ -35,15 +36,15 @@ type Props = {
  *
  * @extends Component
  */
-export default class AbstractMuteRemoteParticipantDialog
-    extends Component<Props> {
+export default class AbstractMuteRemoteParticipantDialog<P:Props = Props>
```

Comment:
Why is this change necessary?

Reply:
Needed to extend the abstract Props provided by `components/AbstractMuteRemoteParticipantDialog` - other uses didn't need this - it's similar to what is being done in `react/features/base/toolbox/components/AbstractButton.js` but with a default to abstract Props so there is no change needed in other places